### DISCUSSION TITLE
🧐🤝 Look into managing grants through platforms such as Opencollective

### DISCUSSION THREAD

**Discussion by ghost** [OWASP/www-committee-project#37] [Ideas]
Open Collective could be beneficial for OWASP in the following ways:

1. Transparent fundraising: OWASP can raise money with full transparency, building trust and accountability within the community.
2. Shared account management: Open Collective allows OWASP to manage their budget collectively, ensuring no single individual takes on all the responsibility while promoting trust and accountability.
3. Powerful money tools: OWASP can use Open Collective to pay expenses, receive donations, manage grants, sell event tickets, get a virtual card linked to their balance, and more.
4. Community engagement: Open Collective provides features for posting updates, creating events, and connecting websites, helping OWASP engage with its community more effectively.
5. Simplified reporting and accounting: Open Collective eliminates the need for messy spreadsheets, making reporting easy and accessible on the platform. This can be especially useful for auditors and grantmakers.
6. Overall, Open Collective can help OWASP streamline its financial operations, foster transparency, and enhance community engagement.

There have also been some efforts to create a funding platform / project to manage this that would be like an OWASP project

**Comment by kingthorin** [comment]
Open? 😉

**Comment by kingthorin** [comment]
@DonnieBLT is the title correct is it really "ope" not "open"? :wink:

**Comment by ghost** [comment]
@kingthorin thank you!

**Comment by thc202** [comment]
OWASP is already in Open Collective https://opencollective.com/owasp I don't know what the current state is though (I seem to recall it was not the path forward at the time).
